src/illumidesk/authenticators/authenticator.py

```diff
@@ -72,10 +72,10 @@ async def setup_course_hook(
     # register the user (it doesn't matter if is a student or instructor) with her/his lms_user_id in nbgrader
     await jupyterhub_api.add_user_to_nbgrader_gradebook(course_id, username, lms_user_id)
     # TODO: verify the logic to simplify groups creation and membership
-    if user_role == 'Student' or user_role == 'Learner':
+    if user_role in ('Student', 'Learner'):
```

Comment:
Should we add these sets as one of the constants to make them more flexible? Eventually, they could reside in a dynamic data store or configuration file.

Reply:
I added a constant called `DEFAULT_ROLE_NAMES_FOR_STUDENT` with both values and a helper method to validate is a user role corresponds to a Student